components/ContentRenderer/components/Attachment/utils.ts

```diff
@@ -13,3 +15,96 @@ export function formatBytes(bytes: number | string): string {
 
     return `${n.toFixed(n >= 10 || l < 1 ? 0 : 1)} ${UNITS[l]}`;
 }
+
+/*
+ * This list contains all mime types ever used in Prezly story content "file attachment" blocks.
+ * Except unrecognized "octet-stream" and "x-download" fallbacks.
+ *
+ * ```sql
+ * SELECT node->'file'->>'mime_type', COUNT(*) FROM press_release, jsonb_array_elements(slate_content_json->'children') node
+ * WHERE node->>'type' = 'attachment'
+ * GROUP by 1
+ * ORDER by 2 DESC;
+ * ```
+ */
+const MAPPING: Record<string, string | undefined> = {
```

Comment:
Maybe it's better to use dedicated [mime-type parsing library ](https://www.npmjs.com/package/mime-types). So that if new mime type appears in some story it would work without updating the theme.
This is used on back-end side anyways, so we should not be concerned about lib size (I haven't checked it)

Reply:
>  This is used on back-end side anyways, so we should not be concerned about lib size (I haven't checked it)

Nope. 

![image](https://github.com/user-attachments/assets/ccf15e95-da22-44ef-b60a-a15ec68a75f0)

I also think it's worth keeping it in-sync with the editor and email renderer, which do use this function.